src/OrchardCore.Modules/OrchardCore.AutoSetup/Options/TenantSetupOptions.cs

```diff
@@ -153,6 +158,11 @@ public virtual IEnumerable<ValidationResult> Validate(ValidationContext validati
             {
                 yield return new ValidationResult(String.Format(RequiredErrorMessageFormat, nameof(SiteTimeZone)));
             }
+
+            if (!String.IsNullOrWhiteSpace(DatabaseSchema) && DatabaseSchema.Any(c => !Char.IsLetterOrDigit(c)))
+            {
+                yield return new ValidationResult("The 'DatabaseSchema' should only contain alpha-numeric chars.");
```

Comment:
```suggestion
                yield return new ValidationResult($"The '{nameof(DatabaseSchema)}' should only contain alpha-numeric chars.");
```

Reply:
Because not yet done in so many other related places